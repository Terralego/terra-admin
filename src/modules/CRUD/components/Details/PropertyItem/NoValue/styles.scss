.CRUD-no-value {
  color: #aeb6bb;;
  font-style: italic;
}
