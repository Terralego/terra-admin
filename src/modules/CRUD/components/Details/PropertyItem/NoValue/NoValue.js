import React from 'react';
import PropTypes from 'prop-types';
import './styles.scss';

const NoValue = ({ t }) => (
  <span className="CRUD-no-value">{t('CRUD.details.noValue')}</span>
);

NoValue.propTypes = {
  t: PropTypes.func,
};

NoValue.defaultProps = {
  t: text => text,
};

export default NoValue;
