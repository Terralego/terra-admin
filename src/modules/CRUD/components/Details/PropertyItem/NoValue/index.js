import { withTranslation } from 'react-i18next';
import compose from '../../../../../../utils/compose';

import NoValue from './NoValue';

export default compose(
  withTranslation(),
)(NoValue);
