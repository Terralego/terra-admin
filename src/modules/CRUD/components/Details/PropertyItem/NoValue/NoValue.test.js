import React from 'react';
import renderer from 'react-test-renderer';
import NoValue from './NoValue';

it('should render correctly', () => {
  const tree = renderer.create((
    <NoValue />
  )).toJSON();
  expect(tree).toMatchSnapshot();
});
