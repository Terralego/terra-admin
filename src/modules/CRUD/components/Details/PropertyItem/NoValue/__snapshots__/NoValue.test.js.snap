// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<span
  className="CRUD-no-value"
>
  CRUD.details.noValue
</span>
`;
